tests/ui/derive-arbitrary/empty_enum/empty_enum.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//! Check that Kani can automatically derive an empty Arbitrary enum.
```

Comment:
"cannot" instead of "can"? But see my comment on making those a no-op.

Reply:
I guess it can now.